pkg/csrapproval/csrapproval.go

```diff
@@ -29,7 +31,6 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/cloud-provider-gcp/pkg/csrmetrics"
 	"k8s.io/klog/v2"
-	certutil "k8s.io/kubernetes/pkg/apis/certificates/v1"
```

Comment:
Can we revert changes in csrapproval.go?

It's part of gcp-controller-manager, which has much more dependencies on K/K. I don't want to surprise owners of the `gcp-controller-manager` when the code will no longer update as expected.

Reply:
done